app/controllers/transfer_packages_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TransferPackagesController < ApplicationController
+  def find_sample
+    @navigation_context = NavigationContext.new(nil, params[:context])
+
+    uuid = params[:uuid]
+    full_uuid = uuid.size == 36
+    @samples = Sample
+      .within(@navigation_context.entity, @navigation_context.exclude_subsites)
+      .joins(:sample_identifiers).where(
+        (full_uuid ? "sample_identifiers.uuid = ?" : "sample_identifiers.uuid LIKE concat(?, '%')"),
+        uuid
+      )
+      .order("created_at DESC")
+      .limit(5)
+
+    @samples = check_access(@samples, READ_SAMPLE)
+
+    if full_uuid && @samples.any?(&:is_quality_control?)
+      render json: { error: "Sample #{uuid} is a QC sample and can't be transferred." }
+      return
+    end
```

Comment:
Let's say I'm autocompleting: I type `ae` see my sample and select it. It was a QC sample. I won't get an error?